client/routes/Workflows.vue

```diff
@@ -1,34 +1,48 @@
 <template>
   <section :class="{ workflows: true, loading }">
     <header class="filters">
-      <div class="field workflow-id">
-        <input type="search" class="workflow-id"
-          placeholder=" "
-          name="workflowId"
-          v-bind:value="$route.query.workflowId"
-          @input="setWorkflowFilter" />
-        <label for="workflowId">Workflow ID</label>
-      </div>
-      <div class="field workflow-name">
-        <input type="search" class="workflow-name"
-          placeholder=" "
-          name="workflowName"
-          v-bind:value="$route.query.workflowName"
-          @input="setWorkflowFilter" />
-        <label for="workflowName">Workflow Name</label>
-      </div>
-      <date-range-picker
-        :date-range="range"
-        :max-days="maxRetentionDays"
-        @change="setRange"
-      />
-      <v-select
-        class="status"
-        :value="status"
-        :options="statuses"
-        :on-change="setStatus"
-        :searchable="false"
-      />
+      <template v-if="filterMode === 'advance'">
+        <div class="field query-string">
+          <input type="search" class="query-string"
+            placeholder=" "
+            key="sql-query"
+            name="queryString"
+            v-bind:value="$route.query.queryString"
+            @input="setWorkflowFilter" />
+          <label for="queryString">Query</label>
+        </div>
+      </template>
+      <template v-else>
+        <div class="field workflow-id">
+          <input type="search" class="workflow-id"
+            placeholder=" "
+            name="workflowId"
+            v-bind:value="$route.query.workflowId"
+            @input="setWorkflowFilter" />
+          <label for="workflowId">Workflow ID</label>
+        </div>
+        <div class="field workflow-name">
+          <input type="search" class="workflow-name"
+            placeholder=" "
+            name="workflowName"
+            v-bind:value="$route.query.workflowName"
+            @input="setWorkflowFilter" />
+          <label for="workflowName">Workflow Name</label>
+        </div>
+        <date-range-picker
+          :date-range="range"
+          :max-days="maxRetentionDays"
+          @change="setRange"
+        />
+        <v-select
+          class="status"
+          :value="status"
+          :options="statuses"
+          :on-change="setStatus"
+          :searchable="false"
+        />
+      </template>
+      <a class="toggle-filter" @click="toggleFilter">{{ filterMode === 'advance' ? 'basic' : 'advance' }}</a>
```

Comment:
Change to "advanced" for the button text.

Reply:
Updated